client/src/pages/mapper/Mapper.js

```diff
@@ -70,8 +62,20 @@ function Mapper() {
           tiles: [`${tilesServerEndpoints}/public.treedata/{z}/{x}/{y}.pbf`],
         });
       });
+      mapboxMap.addControl(new mapboxgl.GeolocateControl(geolocate));
```

Comment:
Wouldn't `mapboxMap.addControl(geolocate);` be sufficient if a `GeolocateControl` was stored in `geolocate` above?